static/app/views/alerts/rules/metric/details/metricChartOption.tsx

```diff
@@ -164,12 +165,20 @@ export function getMetricAlertChartOption({
   handleIncidentClick,
   showWaitingForData,
 }: MetricChartData): MetricChartOption {
-  const criticalTrigger = rule.triggers.find(
-    ({label}) => label === AlertRuleTriggerType.CRITICAL
-  );
-  const warningTrigger = rule.triggers.find(
-    ({label}) => label === AlertRuleTriggerType.WARNING
-  );
+  let criticalTrigger: Trigger | undefined;
```

Comment:
Can revert this in case trigger counts are expected to be low.

Reply:
the majority should have only a few but there's some noisy alerts that have hit the cap of 1000